omo/src/os/linux/mod.rs

```diff
@@ -916,18 +917,28 @@ impl Inner {
     ) -> Result<i64, EmulatorError> {
         log::debug!("write({}, {}, {}) pc: {}", fd, buf, count, core.pc()?);
         let data = Memory::read(core, buf, count as usize)?;
-        let size = write(fd, data.as_ptr(), count);
-        if size < 0 {
-            log::warn!(
-                "failed to write ({}, {}, {}): {:?}",
-                fd,
-                buf,
-                count,
-                from_raw_syscall_ret(size)
-            );
+
+        if fd == 1 {
```

Comment:
what makes stdout stderr special here?

Reply:
It's an old fix. haven't tried on your latest update.